genvm/vm.go

```diff
@@ -501,11 +507,14 @@ func parse(
 		return nil, nil, nil, fmt.Errorf("%w: failed to decode method selector %w", core.ErrMalformed, err)
 	}
 	account, err := loader.Get(principal)
-	if err != nil {
+	switch {
+	case errors.Is(err, core.ErrNotFound):
+		account = types.Account{Address: principal}
+	case err != nil:
 		return nil, nil, nil, fmt.Errorf(
-			"%w: failed load state for principal %s - %w",
+			"%w: failed load state for principal %s: %w",
 			core.ErrInternal,
-			principal,
+			principal.String(),
```

Comment:
`fmt.Errorf` calls `String()` with a `%s` directive.

```suggestion
			principal,
```

Reply:
@fasmat found out that formatting didn't use the `Stringer` implementation because `types.Address` also implements `fmt.Formatter` which takes priority. I remove all implementations of `fmt.Formatter` in the code base and reverted this line.